app/main.py

```diff
@@ -2,0 +1,23 @@
+from app.cafe import Cafe
+
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style Issue: It's good to see that you are using type hints in your function definition. It improves code readability and helps to understand what type of arguments are expected and what is the type of the return value.